apps/right-to-rent-check/acceptance/features/tenancy-start-backlink-submit.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const _ = require('lodash');
+const steps = require('../../');
+
+Feature('Given I have submited a valid tenancy and I go back to the property address page');
```

Comment:
This test is important because I was getting some funnier behaviour before where it was allowing non valid tenancy because it was set before